### ISSUE TITLE
 'css selector' is not supported for this session

### ISSUE THREAD

**Issue by Nisthar** [appium/appium#13904]
I am following the code from http://appium.io/docs/en/commands/element/actions/click/

`$('#SomeId').click();`

"webdriverio": "^5.18.5"

**Comment by mykola-mokhnach** [comment]
Please fill out the issue report properly and attach the full server log

**Comment by Nisthar** [comment]
@mykola-mokhnach I can't even get the client code to work. I think the example code is wrong.
Its saying `$ is not defined`

**Comment by mykola-mokhnach** [comment]
How this is related to the server side? I would rather ask at the corresponding client forum/issue tracker

**Comment by Nisthar** [comment]
@mykola-mokhnach Well i got the code from appium site.

**Comment by imurchie** [comment]
> @mykola-mokhnach I can't even get the client code to work. I think the example code is wrong.
> Its saying `$ is not defined`

This is because the snippets in the documentation are not fully runnable. You still need to set up the test correctly. In this case, I would point you to the https://webdriver.io/ documentation, or to webdriverio sample code for Appium (https://github.com/appium/appium/tree/master/sample-code/javascript-webdriverio)

**Comment by daipn1859** [comment]
@Nisthar 
Are you working in the correct context when you execute click action?

**Comment by spartha1** [comment]
HI
I am also getting this error 
InvalidSelectorException: Message: Locator Strategy 'css selector' is not supported for this session
I am using id as a locator here
Do let me know if need any further info on this

**Comment by nickspiers-test** [comment]
I have just set up Appium on my Windows machine and I am trying to test a real Android Mobile phone.  The test I am trying is a simple one by opening up the calculator and clicking on the number 7.  When I run the test, I too am getting the error "Locator Strategy 'css selector' is not supported for this session".   The calculator does open but then I get the error message when trying to click on the number 7.  My code is:-

driver.findElement(By.id("com.sec.android.app.popupcalculator:id/bt_07")).click();

I am using version 1.17.1 for Appium and I have removed the Selenium dependancy in my pom file as other posts have suggested that this sometimes gets confused.

Can anybody help?
